src/test/kotlin/org/opensearch/commons/alerting/CorrelationAlertTests.kt

```diff
@@ -0,0 +1,112 @@
+package org.opensearch.commons.alerting
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.opensearch.common.xcontent.LoggingDeprecationHandler
+import org.opensearch.common.xcontent.XContentHelper
+import org.opensearch.commons.alerting.model.Alert
+import org.opensearch.commons.alerting.model.CorrelationAlert
+import org.opensearch.commons.utils.getJsonString
+import org.opensearch.core.common.bytes.BytesArray
+import org.opensearch.core.common.bytes.BytesReference
+import org.opensearch.core.common.io.stream.InputStreamStreamInput
+import org.opensearch.core.xcontent.NamedXContentRegistry
+import java.time.temporal.ChronoUnit
+
+class CorrelationAlertTests {
```

Comment:
plz add test for serialization roundtrip..

Reply:
updated